src/cli/commands/config/index.ts

```diff
@@ -0,0 +1,25 @@
+import { Testplane } from "../../../testplane";
+import { CliCommands } from "../../constants";
+import logger from "../../../utils/logger";
+
+const { CONFIG: commandName } = CliCommands;
+
+export const registerCmd = (cliTool: typeof commander, testplane: Testplane): void => {
+    cliTool
+        .command(commandName)
+        .description("Lists all browsers from the config")
+        .option("-c, --config <path>", "path to configuration file")
```

Comment:
for `config` command you can pass the configuration file as an argument rather than as an option

Reply:
No, we can't